lib/miq_pglogical.rb

```diff
@@ -145,7 +145,7 @@ def self.default_excludes
 
   def self.save_remote_region(exclusion_list)
     MiqRegion.replication_type = :remote
-    refresh_excludes(YAML.safe_load(exclusion_list))
+    refresh_excludes(YAML.safe_load(exclusion_list)) unless exclusion_list.empty?
```

Comment:
Why is this exclude list empty?

Reply:
Or rather why would it ever be empty unless the user specifically wanted to exclude nothing? (which as @bdunne noted will likely not go well)

Reply:
Okay, after going back over the previous two PRs it looks like [this condition](https://github.com/ManageIQ/manageiq-ui-classic/pull/4628/files#diff-84dbfcd1a7ac9f7442c66dadf41485bbL221) was previously preventing us from calling the refresh_excludes if the exclude list parameter was empty.

I'm not particularly happy that this UI-side concern (dealing with an empty list that shouldn't be treated as an empty list) is being pushed into this method, but I'm not sure how else this can be solved with the new way this is working.

Can you add a comment that says that this is here because the UI passes an empty array for "no changes" so we can rely on the `refresh_excludes` call which is run as a part of `replication_type=`?